lib/airbrake-ruby/notifier.rb

```diff
@@ -30,7 +30,7 @@ class Notifier
     def initialize(user_config)
       @config = (user_config.is_a?(Config) ? user_config : Config.new(user_config))
 
-      unless [@config.project_id, @config.project_key].all?
```

Comment:
No worries! :)
I am referring to the error message itself. 
It tells me, that the two variables are required, when both values were already set.
So I had to look at the code to get to know it has to be an Integer.

Anyway, I am fine with the state right now and also think its just an edge case.
Thanks!